Source/Core/DolphinWX/ControllerConfigDiag.h

```diff
@@ -69,6 +72,30 @@ class ControllerConfigDiag : public wxDialog
 		SConfig::GetInstance().m_GameCubeAdapterThread = event.IsChecked();
 		event.Skip();
 	}
+	void OnGameCubeAdapterScan(wxCommandEvent& event)
+	{
+		SConfig::GetInstance().m_GameCubeAdapter = true;
+		SConfig::GetInstance().m_GameCubeAdapterThread = true;
```

Comment:
We shouldn't be force enabling the thread.  It should be based on the user setting.